silkrpc/commands/txpool_api.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+    Copyright 2020 The Silkrpc Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include "txpool_api.hpp"
+
+#include <algorithm>
+#include <cstring>
+#include <exception>
+#include <iostream>
+#include <map>
+#include <string>
+#include <utility>
+
+#include <boost/endian/conversion.hpp>
+#include <evmc/evmc.hpp>
+#include <silkworm/chain/config.hpp>
+#include <silkworm/common/util.hpp>
+#include <silkworm/common/base.hpp>
+#include <silkworm/execution/address.hpp>
+#include <silkworm/db/util.hpp>
+#include <silkworm/types/receipt.hpp>
+#include <silkworm/types/transaction.hpp>
+
+#include <silkrpc/common/constants.hpp>
+#include <silkrpc/common/log.hpp>
+#include <silkrpc/common/util.hpp>
+#include <silkrpc/core/cached_chain.hpp>
+#include <silkrpc/core/blocks.hpp>
+#include <silkrpc/core/evm_executor.hpp>
+#include <silkrpc/core/evm_access_list_tracer.hpp>
+#include <silkrpc/core/estimate_gas_oracle.hpp>
+#include <silkrpc/core/gas_price_oracle.hpp>
+#include <silkrpc/core/rawdb/chain.hpp>
+#include <silkrpc/core/receipts.hpp>
+#include <silkrpc/core/state_reader.hpp>
+#include <silkrpc/ethdb/bitmap.hpp>
+#include <silkrpc/ethdb/cbor.hpp>
+#include <silkrpc/ethdb/tables.hpp>
+#include <silkrpc/ethdb/transaction_database.hpp>
+#include <silkrpc/json/types.hpp>
+#include <silkrpc/types/block.hpp>
+#include <silkrpc/types/call.hpp>
+#include <silkrpc/types/filter.hpp>
+#include <silkrpc/types/transaction.hpp>
```

Comment:
I dont' think you need all these inclusion for this file, please try to shrink them to the minimum set

Reply:
ok